tests/test_filters.py

```diff
@@ -17,10 +16,6 @@
 
 normals = ['x', 'y', '-z', (1, 1, 1), (3.3, 5.4, 0.8)]
 
-skip_py2_nobind = pytest.mark.skipif(
-    int(sys.version[0]) < 3, reason="Python 2 doesn't support binding methods"
-)
-
```

Comment:
Oh wow, thank you for finding this... makes me wonder what other Python2 cruft we still have

Reply:
This should be it for the cases easily identified by various `git grep` patterns to find where version checks are used. More complex try/except blocks are trickier to identify, but these should be harmless.